### DISCUSSION TITLE
new mc7455 stuck in low power mode

### DISCUSSION THREAD

**Discussion by ericlee592** [danielewood/sierra-wireless-modems#48] [General]
I just bought an mc7455 went to set it up and it wont work it is stuck in low power mode if any one knows how to fix it could you please let me know. i have tried flashing the firmware and have looked everywhere with no luck. also cant us at!selrat=06 without error

Recieve: ati
Recieve: Manufacturer: Sierra Wireless, Incorporated
Recieve: Model: MC7455
Recieve: Revision: SWI9X30C_02.33.03.00 r8209 CARMD-EV-FRMWR2 2019/08/28 20:59:30
Recieve: IMEI: 359072061513111
Recieve: IMEI SV: 20
Recieve: FSN: LQ911102710210
Recieve: +GCAP: +CGSM
Recieve: OK

Send:at!pcinfo?

Recieve: at!pcinfo?
Recieve: State: Low Power Mode
Recieve: LPM voters - Temp:0, Volt:0, User:0, W_DISABLE:0, IMSWITCH:1, BIOS:0, LWM2M:0, OMADM:0, FOTA:0
Recieve: LPM persistence - None
Recieve: OK

Send:At!priid?

Recieve: At!priid?
Recieve: PRI Part Number: Unknown
Recieve: Revision: Unknown
Recieve: Customer: Unknown
Recieve: Carrier PRI: None
Recieve: OK

Send:at!entercnd=“A710”

Recieve: at!entercnd=“A710”
Recieve: OK

Send:AT!PRIID?

Recieve: AT!PRIID?
Recieve: PRI Part Number: Unknown
Recieve: Revision: Unknown
Recieve: Customer: Unknown
Recieve: Carrier PRI: None
Recieve: OK

Send:at!gobiimpref?

Recieve: at!gobiimpref?
Recieve: !GOBIIMPREF:
Recieve: preferred fw version: 02.33.03.00
Recieve: preferred carrier name: GENERIC
Recieve: preferred config name: GENERIC_002.072_000
Recieve: preferred subpri index: 000
Recieve: current fw version: 02.33.03.00
Recieve: current carrier name:
Recieve: current config name:
Recieve: current subpri index: 000
Recieve: carrier name mismatch
Recieve: config name mismatch
Recieve: OK

**Comment by rdinse** [comment]
I has a similar issue trying to update the firmware on my EM7455 modem where it was stuck in LP mode after failed firmware updates. I was eventually able to update using an old Windows 7 Dell PC, but I do not know which steps I described in [this comment](https://github.com/danielewood/sierra-wireless-modems/issues/52#issuecomment-698548507) in #52 are actually necessary.
